geoutils/misc.py

```diff
@@ -154,6 +155,75 @@ def new_func(*args, **kwargs):  # type: ignore
     return deprecator_func
 
 
+def diff_environment_yml(fn_env: str, fn_devenv: str, print_dep: str = "both") -> None:
```

Comment:
Ok see my comment in the related issue. You can try, but this was probably not needed 😥